package.json

```diff
@@ -26,20 +26,26 @@
   },
   "main": "./lib/ffi",
   "dependencies": {
-    "bindings": "^1.3.0",
     "debug": "^3.1.0",
     "get-uv-event-loop-napi-h": "^1.0.5",
     "node-addon-api": "1.6.1",
+    "node-gyp-build": "^4.2.1",
     "ref-napi": "^1.4.0",
     "ref-struct-di": "^1.1.0"
   },
   "devDependencies": {
+    "bindings": "^1.3.0",
     "fs-extra": "^5.0.0",
     "mocha": "^5.0.0",
-    "nyc": "^12.0.1",
+    "nyc": "^15.0.0",
+    "prebuildify": "^3.0.4",
+    "prebuildify-ci": "^1.0.5",
     "ref-array-di": "^1.2.1"
   },
   "scripts": {
+    "install": "node-gyp-build",
+    "prebuild": "prebuildify --napi",
```

Comment:
You can add `--strip` (strip symbols) to shave off a few bytes.

Reply:
I’d prefer not to do that.

Reply:
How so? Just curious, I personally don't care about prebuild and thus package size.

Reply:
It makes debugging harder, and it’s most likely not saving a lot of space given that this isn’t a large addon to begin with. But mostly I’d like to have a decent debugging experience, because this is the kind of addon that people are bound to make crash… :smile: